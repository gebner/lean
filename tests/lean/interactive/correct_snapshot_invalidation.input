{"seq_num": 0, "command": "sync", "file_name": "f", "content": "inductive foo : Type"}
{"seq_num": 1, "command": "sync", "file_name": "f", "content": "inductive foo : Type\n| bar : foo"}
{"seq_num": 2, "command": "sync", "file_name": "f", "content": "inductive foo : Type\n| bar : foo\nd"}
{"seq_num": 3, "command": "sync", "file_name": "f", "content": "inductive foo : Type\n| bar : foo\ndef"}
{"seq_num": 4, "command": "sync", "file_name": "f", "content": "inductive foo : nat → nat → Prop"}
{"seq_num": 5, "command": "sync", "file_name": "f", "content": "inductive foo : nat → nat → Prop\n"}
{"seq_num": 6, "command": "sync", "file_name": "f", "content": "inductive foo : nat → nat → Prop\n\n"}
{"seq_num": 7, "command": "sync", "file_name": "f", "content": "inductive foo : nat → nat → Prop\n\n\n"}
{"seq_num": 8, "command": "sync", "file_name": "f", "content": ""}
{"seq_num": 9, "command": "sync", "file_name": "f", "content": "import system.io\n"}
