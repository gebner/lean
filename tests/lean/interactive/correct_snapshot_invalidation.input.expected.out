{"message":"file invalidated","response":"ok","seq_num":0}
{"message":"file invalidated","response":"ok","seq_num":1}
{"msgs":[{"caption":"","file_name":"f","pos_col":0,"pos_line":3,"severity":"error","text":"unknown identifier 'd'"}],"response":"all_messages"}
{"message":"file invalidated","response":"ok","seq_num":2}
{"msgs":[{"caption":"","file_name":"f","pos_col":0,"pos_line":3,"severity":"error","text":"unknown identifier 'd'"}],"response":"all_messages"}
{"msgs":[],"response":"all_messages"}
{"msgs":[{"caption":"","file_name":"f","pos_col":3,"pos_line":3,"severity":"error","text":"invalid declaration, identifier expected"}],"response":"all_messages"}
{"message":"file invalidated","response":"ok","seq_num":3}
{"msgs":[{"caption":"","file_name":"f","pos_col":3,"pos_line":3,"severity":"error","text":"invalid declaration, identifier expected"}],"response":"all_messages"}
{"msgs":[],"response":"all_messages"}
{"message":"file invalidated","response":"ok","seq_num":4}
{"message":"file invalidated","response":"ok","seq_num":5}
{"message":"file invalidated","response":"ok","seq_num":6}
{"message":"file invalidated","response":"ok","seq_num":7}
